src/meeting/teams/htmlCleaner.ts

```
@@ -0,0 +1,154 @@
+
+    public async start(): Promise<void> {
+        console.log('[Teams] Starting HTML cleaner')
+
+        // Wait 1 second like in original extension
+        await this.page.waitForTimeout(1000)
+
+        // Inject Teams provider logic into browser context
+        await this.page.evaluate(async (recordingMode) => {
+            // EXACT TEAMS PROVIDER FUNCTIONS FROM ORIGINAL EXTENSION
+            function getDocumentRoot(): Document {
+                for (let iframe of document.querySelectorAll('iframe')) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'iframe' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)